adapter/service/chargepoint/specification.go

```diff
@@ -16,8 +16,9 @@ func Specification(
 	address string,
 	groups,
 	supportedStates []string,
+	supportedChargingModes []string, // Optional
```

Comment:
See comment for `normalizeChargingMode` method.

Reply:
Resolved.